.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,26 @@
+name: Create Release Notes
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
not a blocker for this PR, and potentially should just be done in a follow-up PR:
would be nice to set this up so that it runs automatically as part of [the Publish commit](https://github.com/muxinc/elements/commit/d6078b8dac3edb45a4f2e625a3936446caa02595).

Reply:
indeed, we can add it to cd.yml after 1 manual test